components/nimbus/uniffi.toml

```diff
@@ -1,3 +1,3 @@
 [bindings.kotlin]
-package_name = "org.mozilla.experiments.nimbus"
+package_name = "org.mozilla.experiments.nimbus.internal"
```

Comment:
I'm a little perplexed why you added the `internal` to this, actually, I've been wondering if we should even keep this separate from the other components who are all `org.mozilla.appservices.componentName`.  But that's not something I feel strongly enough about to block this on.

Reply:
For the dumbest reasons: uniffi generates a `nimbus.kt` which was colliding with the `Nimbus.kt` file in the same package.